tests/snapshots/tests.contrib.django.test_django_snapshots.test_asgi_500_3x.snap

```diff
@@ -0,0 +1,369 @@
+[[{"name" "asgi.request"
```

Comment:
Really weird!

In Python 3.7 and above the ASGI middleware + Django 3 generates two traces when exceptions go unhandled. For Python 3.6 this seems to be generally the case _except_ for Django "latest" in which it only generates one... 🤯 

The first one is a 301 redirect which is then followed by the actual 500.


Going to look into why the redirect is being generated....

Reply:
CircleCI pipeline: https://app.circleci.com/pipelines/github/DataDog/dd-trace-py/10031/workflows/48eb8f87-c64a-486d-abf2-f482a4c37b46/jobs/778503/parallel-runs/2

Reply:
Ah, of course. Was due to the lack of a trailing slash in the URL